programs/svm-spoke/src/instructions/admin.rs

```diff
@@ -20,7 +18,7 @@ use crate::{
 pub struct Initialize<'info> {
     #[account(init, // Use init, not init_if_needed to prevent re-initialization.
               payer = signer,
-              space = DISCRIMINATOR_SIZE + State::INIT_SPACE, // TODO: check that INIT_SPACE is used everywhere
```

Comment:
checked this, no changes needed, so removed the comment just to avoid separate PR for that.